music21/musicxml/m21ToXml.py

```diff
@@ -5216,6 +5216,13 @@ def expressionToXml(self, expression):
         >>> MEX.dump(mxExpression)
         <inverted-turn placement="above" />
 
+        >>> invDelayedTurn = expressions.InvertedTurn(delay=1.)
```

Comment:
style: 1.0 -- never a trailing period to create a float.

Reply:
Done.